components/EditUpdateForm.js

```diff
@@ -141,7 +142,7 @@ class EditUpdateForm extends React.Component {
               data-cy="update-content-editor"
             />
           </Container>
-          {(!collective.isHost || update.isPrivate) && (
+          {(!collective.isHost || update.isPrivate) && !this.props.isChangelog && (
```

Comment:
Good idea to hide this for changelog! Some steps we should check:
- Pick "Update"
- Check "Is private"
- Pick "Changelog entry"
- Submit
- `isPrivate` should be false

Reply:
@Betree : Great catch; I've corrected this. 👍🏽